autoload/SimpylFold.vim

```diff
@@ -0,0 +1,229 @@
+let s:non_blank_regex = '^\s*[^[:space:]#]'
+let s:multiline_def_end_regex = '):$'
+let s:multiline_def_end_solo_regex = '^\s*):$'
+let s:docstring_start_regex = '^\s*[rR]\?\("""\|''''''\)\%(.*\1\s*$\)\@!'
+let s:docstring_end_single_regex = '''''''\s*$'
+let s:docstring_end_double_regex = '"""\s*$'
+let s:import_start_regex = '^\%(from\|import\)'
+let s:import_cont_regex = 'from.*\((\)[^)]*$\|.*\(\\\)$'
+let s:import_end_paren_regex = ')\s*$'
+let s:import_end_esc_regex = '[^\\]$'
+
+" Initialize buffer
+function! SimpylFold#BufferInit() abort
+    if &filetype ==# 'pyrex' || &filetype ==# 'cython'
+        let b:SimpylFold_def_regex = '\v^\s*%(%(class|%(async\s+)?def|cdef|cpdef|ctypedef)\s+\w+)|cdef\s*:'
+    else
+        let b:SimpylFold_def_regex = '\v^\s*%(class|%(async\s+)?def)\s+\w+|if\s+__name__\s*\=\=\s*%("__main__"|''__main__'')\s*:'
+    endif
+
+    if !exists('b:SimpylFold_fold_docstring')
+        let b:SimpylFold_fold_docstring = !exists('g:SimpylFold_fold_docstring') || g:SimpylFold_fold_docstring
+    endif
+    if !exists('b:SimpylFold_fold_import')
+        let b:SimpylFold_fold_import = !exists('g:SimpylFold_fold_import') || g:SimpylFold_fold_import
+    endif
+endfunction
+
+" Calculate indent
+function! s:indent(line) abort
+    let ind = matchend(a:line, '^ *') / &tabstop
```

Comment:
I think the problem I mentioned with class folding is due to this line. I have `tabstop` set to 8, but use 4-space indents. I think you should divide by the value of `softtabstop` anyway. `tabstop` is the "number of spaces that a <Tab> in the file counts for", according to the documentation. That means a literal tab character, not the spaces inserted by pressing the tab key with `expandtab` enabled, which I imagine is usually the case when editing Python.